src/NuGet.Core/NuGet.Configuration/NuGet.Configuration.csproj

```diff
@@ -26,7 +26,7 @@
     <Reference Include="System.Xml.Linq" />
   </ItemGroup>
 
-  <ItemGroup Condition=" '$(TargetFramework)' == '$(NetStandardVersion)' or '$(TargetFramework)' == 'net7.0' or '$(TargetFramework)' == 'net8.0' ">
+  <ItemGroup Condition=" '$(TargetFramework)' == '$(NetStandardVersion)' or '$(TargetFramework)' == 'net7.0' or '$(TargetFramework)' == 'net8.0' or '$(TargetFramework)' == 'net9.0' ">
```

Comment:
We don't really build for all t3 here do we? 

Maybe replacement is in order instead of just adding an extra condition?

I don't feel super strongly about fixing though, cause this is more on us as NuGet.Client rather than something you'd need to figure out.

Reply:
Why not just use `"'$(TargetFrameworkIdentifier)' == '.NETCoreApp' or '$(TargetFrameworkIdentifier)' == '.NETStandard'"` here and in all the other places instead?